packages/astro-theme/components/WasThisHelpful.astro

```diff
@@ -14,13 +14,13 @@ interface Props {
 	<button
 		class="rounded-lg px-4 py-1 text-base border border-stone-700 hover:border-green-400 text-stone-300 hover:text-green-400"
 	>
-		😄 {t("Yes", Astro.currentLocale)}
+		😄 Yes
 	</button>
 
 	<button
 		class="rounded-lg px-4 py-1 text-base border border-stone-700 hover:border-red-400 text-stone-300 hover:text-red-400"
 	>
-		😕 {t("No", Astro.currentLocale)}
```

Comment:
Why?

Reply:
Sorry I thought it might be a bit redundant to translate Yes and No, but I looked up some Japanese companies' documentations and they do translate Yes and No into Japanese as well, thanks for pointing it out, I'll modify it!